data_steward/run_curation_pipeline.sh

```diff
@@ -0,0 +1,207 @@
+#copy mapping tables tables
+echo "tools/table_copy.sh --source_app_id ${app_id} --target_app_id ${app_id} --source_dataset ${ehr_snap_dataset} --source_prefix _mapping_ --target_dataset ${unioned_ehr_dataset} --target_prefix _mapping_"
+tools/table_copy.sh --source_app_id ${app_id} --target_app_id ${app_id} --source_dataset ${ehr_snap_dataset} --source_prefix _mapping_ --target_dataset ${unioned_ehr_dataset} --target_prefix _mapping_
+
+########################################################
+#Combine RDR and Unioned EHR data (step 6)
+########################################################
+cdr="${dataset_prefix}combined${today}"
+version="v0-2-rc3"
+
+export RDR_DATASET_ID="${rdr_dataset}"
+export UNIONED_DATASET_ID="${unioned_ehr_dataset}"
+export EHR_RDR_DATASET_ID="${cdr}"
+export BIGQUERY_DATASET_ID="${unioned_ehr_dataset}"
+
+bq mk --dataset --description "${version} combine_ehr_rdr ${rdr_dataset} + ${unioned_ehr_dataset}" ${app_id}:${cdr}
+
+#Create the clinical tables for unioned EHR data set
+python cdm.py ${cdr}
+
+#Copy OMOP vocabulary to CDR EHR data set
+python cdm.py --component vocabulary ${cdr}
+tools/table_copy.sh --source_app_id ${app_id} --target_app_id ${app_id} --source_dataset ${vocab_dataset} --target_dataset ${cdr}
+
+#Combine EHR and PPI data sets
+python tools/combine_ehr_rdr.py
+
+#Run Achilles
+export BIGQUERY_DATASET_ID="${cdr}"
+export BUCKET_NAME_NYC="test-bucket"
+
+python tools/run_achilles_and_export.py --bucket=${result_bucket} --folder=${cdr}
+
+#Close virtual environment and remove
+deactivate
+
+########################################################
+# De-identify the combined dataset (step 7)
+########################################################
+version="v0-2-rc4"
+cdr_deid="${cdr}_deid"
+
+export BIGQUERY_DATASET_ID="${cdr_deid}"
+
+bq mk --dataset --description "${version} deid ${cdr}" ${app_id}:${cdr_deid}
+
+#Create the clinical tables for unioned EHR data set
+python cdm.py ${cdr_deid}
+
+#Copy OMOP vocabulary to CDR EHR data set
+python cdm.py --component vocabulary ${cdr_deid}
+tools/table_copy.sh --source_app_id ${app_id} --target_app_id ${app_id} --source_dataset ${vocab_dataset} --target_dataset ${cdr_deid}
+
+#The location and care_site tables must be deleted (due to this bug):
+bq rm -f --table ${app_id}:${cdr_deid}.location
+bq rm -f --table ${app_id}:${cdr_deid}.care_site
+
+unset PYTHONPATH
+cd deid
+
+#NOTE: Create a copy of config.json
+
+#------Create de-id virtual environment----------
+set -e
+# create a new environment in directory deid_env
+virtualenv -p $(which python2.7) deid_env
+
+# activate it
+source deid_env/bin/activate
+
+# install the requirements in the virtualenv
+pip install -r requirements.txt
+#------------------------------------------------
+
+cd src
+set -e
+
+python deid.py --i_dataset ${cdr} --table person --o_dataset ${cdr_deid} --config ../test_config.json --log
```

Comment:
We need a way for deid's config to be based on user input. We can just pass it as another parameter or let me know any other suggestions; the only thing that changes in `deid/config.json` is the [path to the keyfile](https://github.com/all-of-us/curation/blob/develop/data_steward/deid/config.json#L3) which this script already accepts as input.

```suggestion
python deid.py --i_dataset ${cdr} --table person --o_dataset ${cdr_deid} --config ${deid_config} --log
```